### PR TITLE
Fix mock and FUI writing

### PR DESCRIPTION
**Pull request by yarg0007**
* Corrected mock index increment (was off by -1)
* Updated iOS logger

### CODE REVIEW

**Comment by yarg0007** [comment]
retest

File: NST/src/main/java/com/ebay/service/logger/platforms/HarLogger.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public Platform getPlatformAssociation() {
 	public void writeMocks(List<ServiceCallCacheData> calls, String testClassName, String testMethodName) {
 		
 		FormatWriterUtil.removeMockFilesMatchingClassAndMethodName(testClassName, testMethodName);
-		int sequenceCounter = 0;
+		int sequenceCounter = 1;
```
Comment (juliandesmarais) [inline NST/src/main/java/com/ebay/service/logger/platforms/HarLogger.java:30]: Why was this changed?

Comment (juliandesmarais) [reply NST/src/main/java/com/ebay/service/logger/platforms/HarLogger.java:30]: Nvm, I see it, might cause some issue with our request index validator on iOS, but its ok

File: NST/src/main/java/com/ebay/nst/NSTServiceTestRunner.java [hunk 2]
```diff
@@ -62,9 +60,9 @@ public default void run(IHookCallBack hookCallback, ITestResult iTestResult) {
 			iTestResult.setThrowable(t);
 			iTestResult.setStatus(ITestResult.FAILURE);
 		}
-		
+
 		// If the test passed write mocks/tests
-		if (iTestResult.getStatus() == ITestResult.SUCCESS) {
+		if (iTestResult.getStatus() == ITestResult.SUCCESS || iTestResult.getStatus() == ITestResult.STARTED) {
```
Comment (juliandesmarais) [inline NST/src/main/java/com/ebay/nst/NSTServiceTestRunner.java:65]: Can you please add a comment explaining why we are checking for OR ITestResult.STARTED ?

**Review by juliandesmarais — APPROVED** [review]
1 request
